### DISCUSSION TITLE
server: grouping multiple asciicasts

### DISCUSSION THREAD

**Discussion by omaraboumrad** [asciinema/discussions#205] [Ideas]
Hello,

It would be nice to have the ability to group own recordings on asciinema.org, sort of like a playlist. It makes related recordings more manageable for the owner and users can jump between relevant recordings more easily.

Thanks!

**Comment by ku1ik** [comment]
I never thought of that but this sounds like a really neat thing to have on asciinema.org. Thanks for the idea.
